common/blst/ckb-lib-secp256k1-blst/Makefile

```diff
@@ -24,12 +24,15 @@ build/libckb-lib-secp256k1-blst.a: build/bls12_381_sighash_all.o build/server-as
 build/bls12_381_sighash_all.o: bls12_381_sighash_all.c build/secp256k1_data_info.h
 	$(CC) $(CFLAGS_BLST) -I ${CKB_SCRIPT_SECP256K1} -I ${CKB_SCRIPT_SECP256K1}/src -D __SHARED_LIBRARY__ -c -o $@ $<
 
+build/blake2b_ref.o: c/blake2b_ref.c
+	gcc -I blake2b.h -c -o $@ $<
```

Comment:
Nit: `-I` should specify directories. It's probably not necessary here.

Reply:
Yes, it should be changed to gcc -c -o $@ $<